micropip/_commands/install.py

```diff
@@ -86,6 +87,13 @@ async def install(
         If ``True``, include pre-release and development versions. By default,
         micropip only finds stable versions.
 
+    index_urls :
+
+        A list of URLs or a single URL to use as the package index.
+        By default, micropip uses the PyPI. The URLs should contain
```

Comment:
It would be good indicating what's the actual URL value when None is used (`"https://pypi.org/pypi/{package_name}/json"`). As in if I want to add another index URL in addition to the PyPI one, how do I do that.

Also,
 - Would it be good to indicate that these URLs need to expose a simple JSON API, aside from PyPI? 
 -  worth adding a sentence about the ordering of this list, indicating that the package is first searched in the first index, then if not found in the second and so on.

Reply:
> Would it be good to indicate that these URLs need to expose a simple JSON API, aside from PyPI?

Right, we should mention exactly what kind of APIs we support.